stream/stream_test.go

```diff
@@ -32,4 +32,15 @@ func TestParseFCS(t *testing.T) {
 	_, err = stream.GetAMI("aarch64", "us-east-2")
 	assert.NotNil(t, err)
 	assert.Contains(t, err.Error(), "stream:stable does not have architecture 'aarch64'")
+
+	a, err := stream.QueryDisk("x86_64", "metal", "iso")
+	assert.Nil(t, err)
+	assert.Equal(t, a.Location, "https://builds.coreos.fedoraproject.org/prod/streams/stable/builds/33.20201201.3.0/x86_64/fedora-coreos-33.20201201.3.0-live.x86_64.iso")
+
+	_, err = stream.QueryDisk("x86_64", "metal", "nosuchthing")
+	assert.NotNil(t, err)
+	_, err = stream.QueryDisk("x86_64", "nonformat", "nosuchthing")
+	assert.NotNil(t, err)
```

Comment:
FWIW we're not checking that these are all distinct error paths.

Reply:
Fixed.